src/settings/toml/manifest.rs

```diff
@@ -288,21 +289,21 @@ impl Manifest {
 
     pub fn get_account_id(&self, environment_name: Option<&str>) -> Result<String> {
         let environment = self.get_environment(environment_name)?;
-        let mut result = self.account_id.to_string();
         if let Some(environment) = environment {
             if let Some(account_id) = &environment.account_id {
-                result = account_id.to_string();
+                return Ok(account_id.to_string());
             }
         }
-        if result.is_empty() {
-            let mut msg = "Your configuration file is missing an account_id field".to_string();
-            if let Some(environment_name) = environment_name {
-                msg.push_str(&format!(" in [env.{}]", environment_name));
-            }
-            anyhow::bail!("{}", &msg)
-        } else {
-            Ok(result)
-        }
+        self.account_id.load().map(String::from)
+        // if let Some(id) = result.0 {
+        //     Ok(id)
+        // } else {
+        //     let mut msg = "Your configuration file is missing an account_id field".to_string();
+        //     if let Some(environment_name) = environment_name {
+        //         msg.push_str(&format!(" in [env.{}]", environment_name));
+        //     }
+        //     anyhow::bail!("{}", &msg)
+        // }
```

Comment:
I wasn't sure whether or not to try and keep this context in the new prompt. It seems unfortunate to lose the suggestion to add the ID to the environment file; maybe I could check `account.maybe_load.is_none()` and still give the message anyway? We don't know anymore if the user *intended* for the ID to be in the environment file or not.

Reply:
We can get rid of it. No need to prompt them if we already have it.